controllers/src/main/scala/cromwell/pipeline/controller/ProjectFileController.scala

```diff
@@ -65,10 +66,16 @@ class ProjectFileController(wdlService: ProjectFileService)(implicit val executi
       path("files" / "configurations") {
         post {
           entity(as[ProjectBuildConfigurationRequest]) { request =>
-            onComplete(wdlService.buildConfiguration(request.projectId, request.projectFile)) {
-              case Success(Left(e))              => complete(StatusCodes.UnprocessableEntity, e.errors)
-              case Success(Right(configuration)) => complete(configuration)
-              case Failure(e)                    => complete(StatusCodes.InternalServerError, e.getMessage)
+            onComplete(
+              wdlService
+                .buildConfiguration(request.projectId, request.projectFilePath, request.version, accessToken.userId)
+            ) {
+              case Success(configuration) => complete(configuration)
+              case Failure(e) =>
+                e match {
+                  case ValidationException(msg) => complete(StatusCodes.UnprocessableEntity, msg)
+                  case _                        => complete(StatusCodes.InternalServerError, e.getMessage)
```

Comment:
```suggestion
              case Failure(ValidationException(msg)) => complete(StatusCodes.UnprocessableEntity, msg)
              case Failure(e) => complete(StatusCodes.InternalServerError, e.getMessage)
```

In this case it's better to have a few duplicated symbols rather than nested nested PM.

Reply:
Fixed.